cmake/ExternalProjects.cmake

```diff
@@ -2,8 +2,6 @@ include(FindGit)
 find_package(Git)
 include (ExternalProject)
 
-include_directories(${THIRD_PARTY_INSTALL_DIR}/include)
```

Comment:
We want to install 3rd party deps in the main `CMAKE_INSTALL_PREFIX` location, but we don't want to include everything in there when building, hence we do more fine-grained includes for each library below.